Update Home.md in Chinese

icewolf00 opened the pull request:
I translated the Home.md document into Chinese, if you think its OK, please help put it on the LUIS document of zh-tw.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

icewolf00 committed:
Update Home.md in Chinese

I translated the Home.md document into Chinese, if you think its OK, please help put it on the LUIS document of zh-tw.
